content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -61,30 +60,36 @@ wget https://kubernetes.io/examples/configmap/ui.properties -O configure-pod-con
 kubectl create configmap game-config --from-file=configure-pod-container/configmap/
 ```
 
-combines the contents of the `configure-pod-container/configmap/` directory
-
-```shell
-game.properties
-ui.properties
-```
-
-into the following ConfigMap:
+The above command packages each file, in this case, `game.properties` and `ui.properties` in the `configure-pod-container/configmap/` directory into the game-config ConfigMap. You can display details of the ConfigMap using the following command:
 
 ```shell
 kubectl describe configmaps game-config
 ```
 
-where the output is similar to this:
+The output is similar to this:
 ```
-Name:           game-config
-Namespace:      default
-Labels:         <none>
-Annotations:    <none>
+Name:         game-config
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
 
 Data
 ====
-game.properties:        158 bytes
-ui.properties:          83 bytes
+game.properties:
+----
+enemies=aliens
+lives=3
+enemies.cheat=true
+enemies.cheat.level=noGoodRotten
+secret.code.passphrase=UUDDLRLRBABAS
+secret.code.allowed=true
+secret.code.lives=30
+ui.properties:
+----
+color.good=purple
+color.bad=yellow
+allow.textmode=true
+how.nice.to.look=fairlyNice
```

Comment:
“Create ConfigMaps from files” also has sample output; should that change too?

Reply:
Done.